### PR TITLE
Issue 311 capitalisation of headings

### PR DESCRIPTION
**Pull request by mlagally**
There were different capitalisation rules being used in different contexts.

I applied consistent rules for titles and figure captions.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2983,7 +2983,7 @@ <h2>Thing and Consumer Roles</h2>
             </p>
         </section>
         <section id="sec-topologies-deployment-scenarios">
-            <h2>WoT System's Topologies and Deployment Scenarios</h2>
+            <h2>Topologies of WoT System and Deployment Examples</h2>
```
Comment (mkovatsc) [inline index.html]: Must be "WoT Systems" (plural), no?

**Comment by mkovatsc** [comment]
What styleguide writes "acting", "using", or "same" in lower case when the heading itself is in capitalized style?

**Comment by mlagally** [comment]
The Chicago Manual of Style:

Chicago Manual of Style Capitalization Rules
Chicago Style is one of the most used and respected headline capitalization methods used in journalism. The rules are fairly standard for title case:

Capitalize the first and the last word.
Capitalize nouns, pronouns, adjectives, verbs, adverbs, and subordinate conjunctions.
Lowercase articles (a, an, the), coordinating conjunctions, and prepositions.
Lowercase the ‘to’ in an infinitive (I want to play guitar).

https://www.chicagomanualofstyle.org/book/ed17/frontmatter/toc.html

**Comment by mkovatsc** [comment]
Ehm... Acting, Using, etc. are verbs in present participle. Same is an adjective (here).

**Comment by mlagally** [comment]
You can use the tool at: https://capitalizemytitle.com to convert to Chicago style.
Let's not lose time with purely editorial discussions at this point ;-)

**Comment by mlagally** [comment]
agreed in Arch call on 9.5.: closing without merging to master, since it is included in CR-candidate-3

**Pull request closed without merging**